contracts/validator-manager/ValidatorMessages.sol

```diff
@@ -117,6 +105,85 @@ library ValidatorMessages {
         return subnetConversionID;
     }
 
+    /**
+     * @notice Packs SubnetConversionData into a byte array.
+     * This byte array is the SHA256 pre-image of the subnetConversionID hash
+     * The message format specification is:
+     *
+     * +-------------------+---------------+---------------------------------------------------------+
+     * | convertSubnetTxID : [32]byte        |                                              32 bytes |
+     * +-------------------+-----------------+-------------------------------------------------------+
+     * |    managerChainID : [32]byte        |                                              32 bytes |
+     * +-------------------+-----------------+-------------------------------------------------------+
+     * |    managerAddress : []byte          |                         4 + len(managerAddress) bytes |
+     * +-------------------+-----------------+-------------------------------------------------------+
+     * |        validators : []ValidatorData |                        4 + len(validators) * 88 bytes |
+     * +-------------------+-----------------+-------------------------------------------------------+
+     *                                       | 72 + len(managerAddress) + len(validators) * 88 bytes |
+     *                                       +-------------------------------------------------------+
+     * And ValidatorData:
+     * +--------------+----------+-----------+
+     * |       nodeID : [32]byte |  32 bytes |
+     * +--------------+----------+-----------+
+     * |       weight :   uint64 |   8 bytes |
+     * +--------------+----------+-----------+
+     * | blsPublicKey : [48]byte |  48 bytes |
+     * +--------------+----------+-----------+
+     *                           |  88 bytes |
+     *                           +-----------+
+     *
+     * @param subnetConversionData The struct representing data to pack into the message.
+     * @return The packed message.
+     */
+    function packSubnetConversionData(SubnetConversionData calldata subnetConversionData)
+        internal
+        pure
+        returns (bytes memory)
+    {
+        bytes memory res = new bytes(92 + subnetConversionData.initialValidators.length * 88);
```

Comment:
From the comment above, it wasn't immediately clear to me why the constant was 92 here rather than 72. Mind adding a comment explaining the address is known to be 20 bytes long?